data/human/human missions.txt

```diff
@@ -4057,7 +4057,7 @@ mission "Mafia Extortion"
 				`	"I don't need your credits, just forget it."`
 					goto zerodeal
```

Comment:
```suggestion
```
Not sure if this is the right place but I don't think we need this line.

Reply:
Yes, it's needed.

Reply:
Why would it be?

Reply:
Because it wouldn't go to the label.

Reply:
I'm confused
Default behavior is to just go down so it seems to me it would?

Reply:
It's not necessary, no; the conversation goes to the next available label. It's not wrong to leave it there, but as a matter of style I usually remove them.

Reply:
Me I'd let it here because it feels more logical for me. It'd be weird to remove it.

Reply:
Unless Arachi wants to go through the entire mission and remove the technically unneccessary indents, it's fine to leave it as it is. No harm in it.

Reply:
Yes.

Reply:
@Arachi-Lover Do you agree with the above?

Reply:
I'll remove them in a bit.